x-pack/legacy/plugins/monitoring/public/directives/main/internal_monitoring.js

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Do we need to use an angular directive here? Can this be a react component?

Reply:
This is basically a react component, I guess I could move it outside and into `/components` folder or something. 

I didn't give it much thought since this warning will be removed in `8.0`, right?

Reply:
Yea that's fair, but it just seems weird to be in here and have custom mount code. I think it will be easier to just put this in the `<Overview>` cluster component